editor/src/components/editor/store/editor-state.ts

```diff
@@ -1043,7 +1043,7 @@ export function mergeStoredEditorStateIntoEditorState(
     return {
       ...editor,
       selectedViews: storedEditorState.selectedViews,
-      mode: storedEditorState.mode ?? EditorModes.selectLiteMode(),
+      mode: storedEditorState.mode ?? EditorModes.selectMode(),
```

Comment:
Looks like this will cause issues with existing projects that happened to be left in `SelectLiteMode`, so please can you add a safety check here so that we can default to `SelectMode` if the `storedEditorState.mode` is unrecognised?

Reply:
that's true, fixed!